### PR TITLE
feat: add Amazon links

### PR DESCRIPTION
**Pull request by geman220**

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -5,6 +5,11 @@ import open from 'open';
 import sendNotification from './notification';
 import {Logger} from './logger';
 
+/**
+ * Send test email
+ */
+sendNotification('test');
+
```
Comment (jef) [inline src/index.ts:12]: Let's leave this one out. #24 covers this, but I'd like to see it do a little more.

```suggestion
```

**Review by jef — CHANGES_REQUESTED** [review]

File: src/store/amazon.ts [hunk 2]
```diff
@@ -0,0 +1,56 @@
+import {Store} from './store';
+
+export const amazon: Store = {
+	cartUrl: '',
+	links: [
+		{
+			brand: 'PNY',
```
Comment (jef) [inline src/store/amazon.ts:7]: Do you mind lowercasing everything here?

File: src/store/amazon.ts [hunk 3]
```diff
@@ -0,0 +1,56 @@
+import {Store} from './store';
+
+export const amazon: Store = {
```
Comment (jef) [inline src/store/amazon.ts:3]:
```suggestion
export const Amazon: Store = {
```

File: src/store/index.ts [hunk 4]
```diff
@@ -3,12 +3,14 @@ import {BAndH} from './bandh';
 import {Evga} from './evga';
 import {NewEgg} from './newegg';
 import {Nvidia} from './nvidia';
+import {amazon} from './amazon';
```
Comment (jef) [inline src/store/index.ts:6]:
```suggestion
import {Amazon} from './amazon';
```

File: src/store/index.ts [hunk 5]
```diff
@@ -3,12 +3,14 @@ import {BAndH} from './bandh';
...
 import {Config} from '../config';
 
 const list = new Map([
 	['bestbuy', BestBuy],
 	['bandh', BAndH],
 	['evga', Evga],
+	['amazon', amazon],
```
Comment (jef) [inline src/store/index.ts:13]:
```suggestion
	['amazon', Amazon],
```

**Review by jef — CHANGES_REQUESTED** [review]

File: src/store/amazon.ts (hunk 2 above)
Comment (geman220) [reply src/store/amazon.ts:7]: nope

**Comment by jef** [comment]
Do you mind updating the strings? Looks like there still cased.

**Review by geman220 — COMMENTED** [review]
fixed case

**Review by jef — APPROVED** [review]
Looks great! 💯 Thank you very much!

**Pull request merged**